src/main/java/org/graylog2/gelfclient/transport/GelfUdpTransport.java

```diff
@@ -81,12 +81,12 @@ protected void channelRead0(ChannelHandlerContext ctx, DatagramPacket msg) throw
 
                             @Override
                             public void channelActive(ChannelHandlerContext ctx) throws Exception {
-                                senderThread.start(ctx.channel());
+                                senderThreadReference.get().start(ctx.channel());
```

Comment:
Same as in the TCP transport. I think we should use a local `senderThread` variable here instead of using the reference.